src/platform/tls_schannel.c

```diff
@@ -1176,7 +1176,7 @@ CxPlatTlsSecConfigCreate(
         Credentials->dwFlags |= SCH_CRED_DEFERRED_CRED_VALIDATION;
     }
     if (IsClient) {
-        Credentials->dwFlags |= SCH_CRED_NO_DEFAULT_CREDS;
+        Credentials->dwFlags |= SCH_CRED_NO_DEFAULT_CREDS | SCH_CRED_REVOCATION_CHECK_END_CERT;
```

Comment:
Is this a flag we would ever want to turn off?  OCSP can sometimes slow down client connection times (because it starts another connection to do the OCSP validation)

Reply:
This mimics the behavior of Windows HTTP. **If** we ever need to allow the app to disable this, they can ask for the feature and we can add a new "disable" flag for it.

Reply:
That's extremely confusing to have server default disabled and client default enabled. Couldn't we just use the existing flag for both server and client mode?

Reply:
Discussing over email.